src/components/scenes/Fio/FioNameConfirmScene.tsx

```diff
@@ -11,19 +11,29 @@ import { fioMakeSpend, fioSignAndBroadcast } from '../../../util/FioAddressUtils
 import { logEvent, TrackingEventName, TrackingValues } from '../../../util/tracking'
 import { SceneWrapper } from '../../common/SceneWrapper'
 import { FioActionSubmit } from '../../FioAddress/FioActionSubmit'
+import { withWallet } from '../../hoc/withWallet'
 import { ButtonsModal } from '../../modals/ButtonsModal'
 import { Airship, showError } from '../../services/AirshipInstance'
 import { cacheStyles, Theme, ThemeProps, withTheme } from '../../services/ThemeContext'
 import { SceneHeader } from '../../themed/SceneHeader'
 import { CardUi4 } from '../../ui4/CardUi4'
 import { RowUi4 } from '../../ui4/RowUi4'
 
+export interface FioConfirmParams {
```

Comment:
This should be `FioNamConfirmParams`, to match the filename.

Reply:
This file serves two different scenes `fioDomainConfirm` and `fioNameConfirm`  so I chose a common name

Reply:
I guess this is fine since fio kind of conflates domains and names in their API